frontend/vendor/js/diff.min.js

```diff
@@ -0,0 +1,307 @@
+/*!
+
+ diff v3.1.0
```

Comment:
Can you add this to [package.json](https://github.com/exercism/exercism.io/blob/master/package.json) instead of `vendor`? This way we won't need to keep this file in git and will be able to manage/upgrade versions better.